docs/using.rst

```diff
@@ -412,33 +417,34 @@ Overriding the attribute protocol
 Subclasses which override the attribute-management methods provided by
 :class:`persistent.Persistent`, but must obey some constraints:
 
-:meth:`__getattribute__``
+
+``__getattribute__``
```

Comment:
I don't know from that tool.  It looks as though `:method:` is the [currenly-in-vogue spelling](http://www.sphinx-doc.org/en/stable/domains.html#directive-py:method).